### ISSUE TITLE
Snippets block is incorrectly escaped

### ISSUE THREAD

**Issue by pelson** [KyleKing/mdformat-mkdocs#34]
Given the use of snippets (https://facelessuser.github.io/pymdown-extensions/extensions/snippets/), the documented markup (`--8<-- "filename.ext"`) is escaped, such that it becomes `--8\<-- "filename.ext"`) and the resulting page is the escaped literal, instead of the included snippet.

Tested with the latest version of mdformat-mkdocs

**Comment by pelson** [comment]
Would it be helpful if I try to fix this? I'll be honest and say that I'm not clear at all where such a fix would go (there is no obvious escaping section in the codebase)...

**Comment by KyleKing** [comment]
Thanks, if you have time, I would be open to contributions!

My plan was to add an inline parser that recognizes snippets. The closest recent example I have is here: https://github.com/KyleKing/mdformat-obsidian/pull/6/commits/135790abf683660166a027508233e2167c9e25e0 (before I realized there was an `mdit-py-plugin`)

**Comment by KyleKing** [comment]
I released support for Snippets with v4.1.0!

One note is that the change will only handle formatting for now, which should address your needs. I'm open to contributions if that additional support for complete HTML output is necessary

**Comment by AndyLHCb** [comment]
Hi, @KyleKing I'm afraid this is mostly working well but inline snippets within back-ticks are formatted incorrectly

e.g.

```
`--8<-- "somesnippet.sh"
`
```

is correct for getting an inline snippet with 'inline code' formatting, however this gets corrected to

```
`--8<-- "somesnippet.sh" `
```

which then renders incorrectly.

```
`
--8<-- "somesnippet.sh"
`
```

would also display correctly
